.github/workflows/testing.yaml

```diff
@@ -33,10 +33,10 @@ jobs:
 
```

Comment:
The `test/var/memcached` symlink should be removed together with test-run update, I think, because otherwise it is deleted by a first in-source run of tests: it is considered by `git status` as a tracked file change.

Reply:
done